openupgrade_scripts/scripts/account/15.0.1.2/upgrade_analysis.txt

```diff
@@ -13,7 +13,6 @@ account      / account.account.template / message_ids (one2many)        : NEW re
 account      / account.account.template / message_main_attachment_id (many2one): NEW relation: ir.attachment
 account      / account.account.template / website_message_ids (one2many): NEW relation: mail.message
 account      / account.analytic.line    / category (False)              : NEW selection_keys: ['invoice', 'other', 'vendor_bill'], mode: modify
-account      / account.bank.statement.line / narration (text)              : type is now 'html' ('text')
```

Comment:
@tranngocson1996 không nên xóa dòng này nhé, để lại và ghi là `# NOTHING TO DO: inherited from account.move`